charts/capsule-proxy/templates/ingress.yaml

```diff
@@ -31,10 +31,20 @@ spec:
     http:
       paths:
       {{- range .paths }}
+      {{- if semverCompare ">=1.12-0" $.Capabilities.KubeVersion.GitVersion }}
```

Comment:
Actually, were expecting Capsule to be running on Kubernetes >= 1.16 due to certain constraints, mostly regarding API versions and some admission controllers.

Although the proposed change is optimal, what do you think about putting the said version as check?

Reply:
I actually changed it to checking for the availability of `networking.k8s.io/v1/Ingress` instead which is present as of k8s v1.19 :+1: